projects/rero/ng-core/src/lib/record/record-statistics/record-statistics.component.html

```diff
@@ -0,0 +1,31 @@
+<!--
+  RERO angular core
+  Copyright (C) 2020 RERO
+ 
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+ 
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+ 
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<ng-container>
+    <h5 id="stats" class="mt-5">{{'Statistics' | translate}}</h5>
+    <hr class="mb-4 mt-0"/>
+    <div id="record_stats" class="row">
+        <div class="col">
+            <span class="font-weight-bold">{{'Document views: ' | translate}}</span>{{statistics['record-view']}}
+            <div *ngIf="statistics['file-download']">
+                <span class="font-weight-bold mt-2 d-block">{{'File downloads:' | translate}}</span>
+                <ul class="mb-0" *ngFor="let stat of statistics['file-download'] | keyvalue">
+                    <li>{{ stat.key }}: {{ stat.value }}</li>
+                </ul>
+            </div>
+        </div>
+    </div>
```

Comment:
there are some problem with your code : 
  * according bootstrap doc `row` should not be used without any parant `container` tag.
  * `row` with only a single `col` element is useless.
  * translations should not contains ':'
  * try to use useful HTML tags instead of div/span (here label seems what you want)
  * `ngFor` are not on UL but on LI tags. 

```html
<div class="container">
    <div class="row">
        <label class="col-3">{{ 'Document views' | translate }}:</label>
        <span class="col-9">{{ statistics['record-view'] }}</span>
        <ng-container *ngIf="statistics['file-download']">
            <label class="col-3">{ {'File downloads' | translate }}:</label>
            <ul class="col-9 mb-0">
                <li *ngFor="let stat of statistics['file-download'] | keyvalue">{{ stat.key }}: {{ stat.value }}</li>
            </ul>
        </ng-container>
    </div>
</div>
```

Reply:
@zannkukai  I re-wrote this div following your suggestions, although I did not use col-3 and col-9 because it did not look nice the way it does display on the page. Let me know if the changes I did are fine with you. Thanks.

Reply:
great for me is the look and feel is great for you.